_partials/ubuntu_vscode.md

```diff
@@ -0,0 +1,83 @@
+
+## Visual Studio Code - Your text editor
+
+A text editor is one of the most important tools of a developer.
+Follow these instructions in the Terminal:
+
+```bash
+sudo snap install --classic code
+```
+
+:point_up: This command will ask for your password with: `[sudo] password for <username>:`. Don't panick! Calmly type your password key by key. You won't have a visual feedback (like little `*`), that's **perfectly normal**, keep on typing. When you're done, hit `Enter` :muscle:.
+
+In case you don't have `snap` installed - ask a TA for help.
+
```

Comment:
Extra empty lines are useless because they are not rendered when GitHub converts markdown to HTML. Can you delete them everywhere in your files? :pray:

Reply:
I always keep them for the markdown readability. Agree with the double ones or the empty lines within a block, but I'd keep some to maintain the structure. Wdyt?

Reply:
If it improve readability, yes sure!